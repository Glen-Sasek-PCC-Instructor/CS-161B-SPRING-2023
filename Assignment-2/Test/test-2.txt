d
e
SteVens
MaC
y
234-54-9876
assignment2.cpp
12:23
q
