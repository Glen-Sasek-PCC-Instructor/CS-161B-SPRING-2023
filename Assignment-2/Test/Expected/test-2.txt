Welcome to my fileName encoding program!!

Please pick an option below:
(e) Encode a file name
(q) Quit
>>
Invalid input. Please try again.
>> 
This program will ask you a few questions and generate an encoded fileName based on your
answers.
Enter your last name: 
Enter your first name: 
Was your assignment Late (y/n)? 
Enter your Student-ID (format: 222-22-2222):
Enter the file name: 
Enter the time submitted (military time - ex: 18:24 for 6:24pm): 
Your encoded file name is: stevens_mac_LATE_9876_1223_assignment2.cpp
Please pick an option below:
(e) Encode a file name
(q) Quit
>>
Goodbye!